packages/editor/src/worker/OptimizerWorker.ts

```
@@ -1,308 +1,802 @@
+        }
+
+        case "Optimizing": {
+          await this.interruptOptimizing();
+          const jobId = await this.compile(domain, style, substance, variation);
+          resolve(jobId);
+          break;
+        }
+
+        // exhaustive
+      }
+    });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)